File: javascript/1183.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var valores = input.split('\n'), sum = 0, count = 0, numeros = 0;
var oper = String(valores.shift()).trim();

for (i = 0; i < 12; i++) {
    for (j = 0; j < 12; j++) {
        var val = parseFloat(valores.shift());
        if (j > count) {
            sum += val;
            numeros++;
        }
    }
    count++;
}
console.log(oper == "S" ? sum.toFixed(1) : (sum / numeros).toFixed(1));

File: C/1183.c
#include <stdio.h>

int main(){
    int i,j,count=0;
    double sum=0,matriz,numeros=0;
    char tipo;
    scanf("%c",&tipo);
    
    for(i=0;i<12;i++){
        for(j=0;j<12;j++){
            scanf("%lf",&matriz);
            if(j > count){
                sum+=matriz;
                numeros++;
            }
        }
        count++;
    }
    printf("%.1lf\n", tipo == 'S' ? sum : sum / numeros);
    
    return 0;
}